Update README.md

Timboid opened the pull request:
I'm making this PR for learning purposes, as I want to practise my GitHub skills.

I added an apostrophe for purely grammatical purposes.

I added an example of a page that uses Local Links Manager.

⚠️ This repo is Continuously Deployed: make sure you [follow the guidance](https://docs.publishing.service.gov.uk/manual/development-pipeline.html#merge-your-own-pull-request) ⚠️

Timboid committed:
Update README.md

I added an apostrophe for purely grammatical purposes.

I added an example of a page that uses Local Links Manager.

ChrisBAshton commented on README.md:
```diff
@@ -1,6 +1,8 @@
-Admin interface for managing Local Authorities links including all their services and interactions.
+Admin interface for managing Local Authorities' links including all their services and interactions.
+
+For example, [this service](https://www.gov.uk/garden-waste-disposal) uses Local Links Manager to provide the URL on a Local Authority's website that contains information on disposing of garden waste.
```
I think there may be a typo below:

```suggestion
For example, [this service](https://www.gov.uk/garden-waste-disposal) uses Local Links Manager to provide the URL of a Local Authority's website that contains information on disposing of garden waste.
```

You can click "Commit suggestion" to have GitHub automatically create an additional commit for you. Though we typically prefer rebasing our commits, which may be a Git step too far at the moment!

Timboid replied:
Thanks @ChrisBAshton. I did actually mean 'on', because the Local Links Manager should give you the specific URL, not the homepage (which is what I would understand from 'of'). But I agree it's not quite clear at the moment. How about:

`For example, this service uses Local Links Manager to provide the specific URL, on each Local Authority's website, with information about how to dispose of garden waste.`

Rebasing sounds like a good idea. But might need another pairing session to talk through.

ChrisBAshton replied:
Ah, I see what you mean. And I like your new proposed text, but think it could do with a final tweak to make it clear it's talking about one Local Authority (not multiple):

```suggestion
For example, [this service](https://www.gov.uk/garden-waste-disposal) uses Local Links Manager to determine your Local Authority and links to the garden waste disposal page on the Local Authority's website.
```

Timboid replied:
Thanks @ChrisBAshton. Local Links Manager doesn't actually determine a user's Local Authority - that's what Mapit does. I've removed that from your text, but have added a second sentence to make clear the difference between the two tools. (I'm not mentioning Mapit - our current postcode lookup tool - by name, because we're planning to replace it.)

```
For example, this service uses Local Links Manager to determine the URL of the garden waste disposal page on the Local Authority website which corresponds to the postcode a user has entered.

Note that Local Links Manager does not determine a user's Local Authority based on their postcode. We use a separate postcode lookup tool for that.
```

ChrisBAshton replied:
Looks good 👍 I think it's worth replacing `We use a separate postcode lookup tool for that.` with `We use [Mapit][LINK] for that.` But otherwise I think we're good to go, and have had plenty of practice in the GitHub review process! :)

Timboid replied:
Thanks @ChrisBAshton! Final suggested text:

```
For example, this service uses Local Links Manager to determine the URL of the garden waste disposal page on the Local Authority website which corresponds to the postcode a user has entered.

Note that Local Links Manager does not determine a user's Local Authority based on their postcode. We use [Mapit](https://github.com/alphagov/mapit) for that.

``` 

- Now that I've linked to Mapit in the code, and we know that we're going to replace it, is there a standard place I can make a note of this, so that we know to update this text when we get rid of Mapit?
- I'm not sure how to make another change to my code (but would like to learn). I think a quick pair would be the best for this. I'll ask for another pair on #govuk-platform-health Slack.

Timboid committed:
Update README.md

This commit is primarily a learning process for me.

The commit makes basic updates to the readme intro, based on the comments from @ChrisBAshton on my previous commit.

I have a suspicion I've not quite followed the right process here - apologies and bear with me if so.

ChrisBAshton commented on README.md:
```diff
@@ -2,7 +2,9 @@
 
 Admin interface for managing Local Authorities' links including all their services and interactions.
 
-For example, [this service](https://www.gov.uk/garden-waste-disposal) uses Local Links Manager to provide the URL on a Local Authority's website that contains information on disposing of garden waste.
```
Did you mean to remove this link?

Timboid replied:
I didn't. Thanks. Is there a way I can accept your suggestion, or do I need to make another new commit?

ChrisBAshton replied:
You can accept my suggestion if I mark it up in the comment (using '''suggestion syntax, but with ` instead of '). I'll do that here:

```suggestion
For example, [this service](https://www.gov.uk/garden-waste-disposal) uses Local Links Manager to determine the URL of the garden waste disposal page on the Local Authority website which corresponds to the postcode a user has entered.
```

But as this is a fixup commit (correcting a mistake that you don't really need to retain a history of), it's probably better to follow the CLI instructions for amending the commit and writing over your branch, rather than creating a new commit. Depends how far you'd like to take this learning session!

Timboid replied:
I've discussed with @ChrisBAshton and will make a new commit, as I don't plan to learn how to amend commits at this early stage in my Git learning journey 🛤

Timboid committed:
Update README.md

Adds a hyperlink which was lost by mistake in the previous commit.

As before, this update is primarily a learning process for me.  The reason for creating a new commit each time rather than amending my previous commits is that I don't know how to do that, and don't plan to learn at this early stage in my Git journey.

ChrisBAshton reviewed (approved):
You can now hit Merge 👍
